content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: KIND
```

Comment:
On the referenced kind site, they use kind instead of KIND. I think this needs to be changed globally.

Reply:
I kept it capital to avoid confusion with k8 `kind`. I can update it if it does not look good.